docs/content/en/docs/install/install.md

```diff
@@ -0,0 +1,109 @@
+  the [manifests](#use-manifests).
+  This is the less-preferred way because it does not support customization.
+
+## Use Helm Chart
+
+Version 0.7.0 and later of the Lifecycle Toolkit
+should be installed using Helm Charts.
+The command sequence to fetch and install the latest release is:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Note that the `helm update` command is used for fresh installs
+as well as for upgrades.
+
+Use the `--version <version>` flag on the
+`helm upgrade --install` command line to specify a different KLT version..
+
+Use the following command sequence to see a list of available versions:
+
+```shell
+helm repo update
+helm search repo keptn-lifecycle-toolkit
+```
+
+To modify configuration options, use the `--set` flag
+to the `helm upgrade --install` command.
+Configuration options are specified using the format:
+
+```shell
+key1=value1,key2=value2,....
+```
+
+You can instead download a copy of the
+[helm/chart/values.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/values.yaml)
+file, modify some values, and use the modified file to install KLT:
+
+1. Download the `values.yaml` file:
+
+   ```shell
+   helm get values RELEASE_NAME [flags] > values.yaml
+   ```
+
+1. Edit your local copy to modify some values
+
+1. Install KLT by adding the following string to your `helm upgrade` command line:
+
+   ```shell
+   --values=values.yaml
+   ```
+
+For more information,see
+
+* The [Helm Get Values](https://helm.sh/docs/helm/helm_get_values/)) document
+
+* The [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/README.md) page
+  contains the full list of available flags.
+
+## Use manifests
+
+All versions of the Lifecycle Toolkit can be installed using manifests,
+although we recommend that you use Helm Charts
+to install Version 0.7.0 and later
+because the Helm Charts allow you to customize your configuration.
+
+Versions 0.6.0 and earlier can only be installed using manifests.
+
+**Note:** When installing Version 0.6.0,
+you must first install the `cert-manager` with the following command sequence:
+
+```sh
+kubectl apply\
+   -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+kubectl wait\
+   --for=condition=Available deployment/cert-manager-webhook -n cert-manager --timeout=60s
+```
+
+Use a command sequence like the following
+to install the Lifecycle Toolkit from the manifest,
+specifying the version you want to install.
+
+```shell
+kubectl apply \
+   -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0.6.0/manifest.yaml
+kubectl wait --for=condition=Available deployment/lifecycle-operator\
+   -n keptn-lifecycle-toolkit-system --timeout=120s
+```
+
+The Lifecycle Toolkit and its dependencies are now installed and ready to use.
```

Comment:
We should explain that KLT installs as an Operator (or Operators [see below]?) into the `keptn-lifecycle-toolkit-system` namespace.

Link to the kubernetes operator page so readers can find out more if they're unfamiliar: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/

In fact, I think KLT may now consist of two separate operators but I can't find any docs on that and the readmes are blank. Needs dev input: https://github.com/keptn/lifecycle-toolkit/blob/main/operator/README.md

The page should then explain what the operator is doing (which I can't find that info in the readmes). Needs dev input: https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/README.md

If the page is explaining why operator(s) are installed and what they do, the page should also explain why a cert-manager is installed, what it does and how the use of a user-supplied cert-manager can be used instead: https://github.com/keptn/lifecycle-toolkit/issues/1117

All of that said, perhaps all that technical detail is better elsewhere but just "linked to"?

Can a sentence and link be added at the end of this document so that the user can now "flow" into "ok, it's installed, how do I use it"?

Note: none of these review comments are meant to be read as blockers to this PR. FWIW I fully support a quick merge of this PR. IMO incremental, quick improvements are better than a "wait for perfection / big bang" approach. So FWIW this gets the 👍from me.

Reply:
Yes, the point of this PR is to pull together existing material into a section so we can start to xref it from other docs.  We will then come back and enhance the information.  Created https://github.com/keptn/lifecycle-toolkit/issues/1169 for this issue.